ui/src/pageLayout/components/NavMenu.tsx

```diff
@@ -9,12 +9,23 @@ import NavMenuSubItem from 'src/pageLayout/components/NavMenuSubItem'
 // Types
 import {ErrorHandling} from 'src/shared/decorators/errors'
 
-interface Props {
+interface PassedProps {
   children: JSX.Element[]
+  className?: string
 }
 
+interface DefaultProps {
+  testID?: string
+}
+
+type Props = PassedProps & Partial<DefaultProps>
```

Comment:
@121watts did some work with the default props pattern here influxdata/influxdb#12956. in order to follow that pattern, `testID` should be just a required prop in passed props and typescript will interpret the type of the defaultProps.